templates/python/.github/workflows/pre-commit.yaml

```
@@ -0,0 +1,52 @@
+name: Pre-commit
+
+on:
+  push:
+    # branches: [master]
+    tags: ["v[0-9]+.[0-9]+.[0-9]+"]
+  pull_request:
+    branches: [master]
+
+jobs:
+  pre-commit:
+    name: pre-commit
+    runs-on: ubuntu-latest
+    env:
+      working-directory: ./
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version: ["3.8"]
```

let's use something more recent as default
```suggestion
        python-version: ["3.10", "3.11"]
```

---

fyi it should always match what's specified in `pyproject.toml` as Python depedency

---

done